From 0e1830f618187a91c51262a8540ea5cc6398557a Mon Sep 17 00:00:00 2001
From: Lee Jenfeng <jenfeng.lee@cortina-access.com>
Date: Thu, 20 Aug 2020 11:25:13 +0800
Subject: [PATCH 12/15] System Integration in Saturn2

Although not all features are ready, U-Boot could boot kernel from flash
successfully.
---
 configs/cortina_saturn2-engboard-nand_defconfig | 5 +++++
 configs/cortina_saturn2-engboard-nor_defconfig  | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/configs/cortina_saturn2-engboard-nand_defconfig b/configs/cortina_saturn2-engboard-nand_defconfig
index b012ea79cb..2de7ce1a6d 100644
--- a/configs/cortina_saturn2-engboard-nand_defconfig
+++ b/configs/cortina_saturn2-engboard-nand_defconfig
@@ -59,3 +59,8 @@ CONFIG_CORTINA_NI_ENET=y
 CONFIG_NET=y
 CONFIG_CMD_NET=y
 CONFIG_CMD_PING=y
+
+# System Integration
+CONFIG_HUSH_PARSER=y
+CONFIG_CMD_CORTINA_IMAGE_SELECT=y
+CONFIG_LZMA=y
diff --git a/configs/cortina_saturn2-engboard-nor_defconfig b/configs/cortina_saturn2-engboard-nor_defconfig
index 56cdcda6d9..f0d1e584bc 100644
--- a/configs/cortina_saturn2-engboard-nor_defconfig
+++ b/configs/cortina_saturn2-engboard-nor_defconfig
@@ -67,3 +67,8 @@ CONFIG_CORTINA_NI_ENET=y
 CONFIG_NET=y
 CONFIG_CMD_NET=y
 CONFIG_CMD_PING=y
+
+# System Integration
+CONFIG_HUSH_PARSER=y
+CONFIG_CMD_CORTINA_IMAGE_SELECT=y
+CONFIG_LZMA=y
-- 
2.20.1

